genai-perf/genai_perf/inputs/file_input_retriever.py

```diff
@@ -182,9 +157,16 @@ def _get_prompts_from_input_file(self) -> Tuple[List[str], List[str]]:
         with open(self.config.input_filename, mode="r", newline=None) as file:
             for line in file:
                 if line.strip():
+                    data = load_json_str(line)
                     # None if not provided
-                    prompt = load_json_str(line).get("text_input")
-                    image = load_json_str(line).get("image")
+                    prompt = data.get("text_input")
+                    prompt_alt = data.get("text")
+                    if prompt and prompt_alt:
+                        raise ValueError(
+                            "Each data entry must have only one of 'text_input' or 'text' key name."
+                        )
```

Comment:
I modified `load_json_str` to perform checks like this. You can do something like:
```python
def __check_keyname(x, keys):
    if "text_input" in x and "text" in x:
        raise ValueError(
            "Each data entry must have only one of 'text_input' or 'text' key name."
         )

data = load_json_str(line, func=__check_keyname)
```

Reply:
@nv-hwoo What's the benefit of doing it this way? It looks to have the same outcome but add an extra function definition.

Reply:
I just think it's more explicit (and as a result more clear) in terms of what we are checking. The function is directly checking whether the keywords are in the json dictionary, whereas the current if statement is checking the outcome of the `get` method.

It's a nit and a suggestion, so no need to be gated by this.

Reply:
Thanks for explaining. I'll merge this now, but if we want to think it's clearer in the future, then we can change the approach.